diff -uNr ncurses-6.1-20181117/progs/tic.c ncurses-6.1-20181117.mod/progs/tic.c
--- ncurses-6.1-20181117/progs/tic.c	2018-03-18 02:05:10.000000000 +0200
+++ ncurses-6.1-20181117.mod/progs/tic.c	2019-03-01 20:40:56.193173489 +0200
@@ -386,7 +386,7 @@
 {
     FILE *result = 0;
 
-    _nc_STRCPY(filename, "/tmp/XXXXXX", PATH_MAX);
+    _nc_STRCPY(filename, "/data/local/tmp/XXXXXX", PATH_MAX);
 #if HAVE_MKSTEMP
     {
 	int oldmask = (int) umask(077);
