diff -u -r /home/builder/lib/android-ndk-r28b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/paths.h ./usr/include/paths.h
--- /home/builder/lib/android-ndk-r28b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/paths.h	2025-04-14 22:43:37.000000000 +0000
+++ ./usr/include/paths.h	2025-05-09 18:45:44.121425904 +0000
@@ -61,3 +61,9 @@
 
 /** Path to the calling process' tty. */
 #define _PATH_TTY "/dev/tty"
+
+#define        _PATH_STDPATH   _PATH_DEFPATH
+#define        _PATH_TMP       "/data/local/tmp/"
+#define        _PATH_VARDB     "/data/local/var/db/"
+#define        _PATH_VARRUN    "/data/local/var/run/"
+#define        _PATH_VARTMP    "/data/local/var/tmp/"
