diff -u -r pwd.h.orig pwd.h
--- pwd.h.orig
+++ pwd.h
@@ -89,13 +89,55 @@
 struct passwd* _Nullable getpwent(void) __INTRODUCED_IN(26);
 
 void setpwent(void) __INTRODUCED_IN(26);
-void endpwent(void) __INTRODUCED_IN(26);
 #endif /* __BIONIC_AVAILABILITY_GUARD(26) */
 
 
 int getpwnam_r(const char* _Nonnull __name, struct passwd* _Nonnull __pwd, char* _Nonnull __buf, size_t __n, struct passwd* _Nullable * _Nonnull __result);
 int getpwuid_r(uid_t __uid, struct passwd* _Nonnull __pwd, char* _Nonnull __buf, size_t __n, struct passwd* _Nullable * _Nonnull __result);
 
+static __inline__ void endpwent(void) {
+    /* No-op implementation for Android */
+}
 __END_DECLS
 
-#endif
+#endif /* _PWD_H_ */
