--- a/libcontainer/criu_linux.go
+++ b/libcontainer/criu_linux.go
@@ -162,7 +162,7 @@
 	} else {
 		// If the mentioned annotation has not been found, specify
 		// a default CRIU configuration file.
-		rpcOpts.ConfigFile = proto.String("/etc/criu/runc.conf")
+		rpcOpts.ConfigFile = proto.String("/data/docker/runc.conf")
 	}
 }
 
--- a/create.go
+++ b/create.go
@@ -25,7 +25,7 @@
 		},
 		cli.StringFlag{
			 Name:  "bundle, b",
-			Value: "",
+			Value: "/data/docker/runc",
 			Usage: `path to the root of the bundle directory, defaults to the current directory`,
 		},
 		cli.StringFlag{
--- a/run.go
+++ b/run.go
@@ -26,7 +26,7 @@
 		},
 		cli.StringFlag{
 			Name:  "bundle, b",
-			Value: "",
+			Value: "/data/docker/runc",
 			Usage: `path to the root of the bundle directory, defaults to the current directory`,
 		},
 		cli.StringFlag{
--- a/restore.go
+++ b/restore.go
@@ -55,7 +55,7 @@
 		},
 		cli.StringFlag{
 			Name:  "bundle, b",
-			Value: "",
+			Value: "/data/docker/runc",
 			Usage: "path to the root of the bundle directory",
 		},
 		cli.StringFlag{
--- a/spec.go
+++ b/spec.go
@@ -66,7 +66,7 @@
 		},
 		cli.StringFlag{
 			Name:  "bundle, b",
-			Value: "",
+			Value: "/data/docker/runc",
 			Usage: "path to the root of the bundle directory",
 		},
 		cli.BoolFlag{
